internal/commands/mfa.go

```diff
@@ -38,12 +39,22 @@ func (b *commandsBuilder) newMfaCmd() *mfaCmd {
 				return err
 			}
 
+			devices, err := iam.New(sess).ListMFADevices(&iam.ListMFADevicesInput{})
+			if err != nil {
+				return err
+			}
+
+			if len(devices.MFADevices) == 0 {
+				cc.log.Error("MFA doesn’t exist")
+				return fmt.Errorf("MFA doesn’t exist")
```

Comment:
@psihachina "MFA doesn’t exist" - where? probably add more info for user....or probably rename to: "MFA hasn't configured" ?